### PR TITLE
Fix doc abbrev

### PR DESCRIPTION
**Pull request by duncdrum**
I already pushed a release to be able to test this with other apps, if it breaks linking we'll need to revert before releasing the next documentation update.

see eXist-db/exist#2194

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <groupId>org.exist-db</groupId>
     <artifactId>exist-apps-parent</artifactId>
-    <version>1.9-SNAPSHOT</version>
+    <version>1.10-SNAPSHOT</version>
```
Comment (adamretter) [inline pom.xml]: This should never be changed manually. Instead we let the release plugin handle it.

File: pom.xml [hunk 2]
```diff
@@ -54,7 +54,7 @@
         <package-final-name>${project.artifactId}-${project.version}</package-final-name>
 
         <!-- other packages that the documentation (of apps) links to; their `abbrev` must match their expath-pkg.xml -->
-        <documentation.pkg.abbrev>exist-documentation</documentation.pkg.abbrev>
+        <documentation.pkg.abbrev>doc</documentation.pkg.abbrev>
```
Comment (adamretter) [inline pom.xml]: I don't actually see this as a good thing. Let's discuss some more...

File: pom.xml (hunk 1 above)
Comment (duncdrum) [reply pom.xml]: it was release plugin, that handles that, but a release artifact still needs to be pulled from master

File: pom.xml (hunk 2 above)
Comment (duncdrum) [reply pom.xml]: let's not. If this one line fix doesn't solve the problem, we ll need to fix the build script and public-repo code properly. We still have broken links, and code despite having a separate name-my-app-app. Good indication that we're over-engineering hacks, without fixing the underlying problem.

**Comment by adamretter** [comment]
@duncdrum what is the status of this?

**Comment by duncdrum** [comment]
I think the build scripts and public-repo should be fixed by now, which was always the better option

**Pull request closed without merging**